rpc/handlers.go

```diff
@@ -1348,7 +1348,7 @@ func (h *Handler) Call(call FunctionCall, id BlockID) ([]*felt.Felt, *jsonrpc.Er
 	}, &vm.BlockInfo{
 		Header:                header,
 		BlockHashToBeRevealed: blockHashToBeRevealed,
-	}, state, h.bcReader.Network(), h.callMaxSteps)
+	}, state, h.bcReader.Network(), h.callMaxSteps, true)
```

Comment:
I think for v0_6 this should be false, right?

Reply:
Yep, fixed it

Reply:
does is make sense to have a `call` private func to reduce the duplication and pass v0_6Response to it?

Reply:
yep, changed it